site/en/docs/react_gemini_prompting.ipynb

```diff
@@ -0,0 +1,973 @@
+{
```

Comment:
<p>Use noun-phrases in headings - here maybe <code>Define tools</code> would do?</p><p> </p><p>Good advice in here: <a href="https://developers.google.com/style/headings" target="_blank">https://developers.google.com/style/headings</a> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>